src/ecalc/libraries/libecalc/common/tests/integration/snapshots/test_consumer_system_v2/test_compressor_system_v2_results/consumer_system_v2.json

```diff
@@ -77,23 +147,23 @@
                         "2022-01-01 00:00:00",
                         "2026-01-01 00:00:00"
                     ],
-                    "typ": "CALENDAR_DAY",
-                    "unit": "Sm3/d",
+                    "typ": "STREAM_DAY",
+                    "unit": "MW",
                     "values": [
-                        39000.0,
-                        39000.0
+                        0.0,
+                        0.0
                     ]
                 },
-                "id": "b16b9265fc6f9743594a2441fcc0d7c8",
+                "id": "9ecb7728acb755a3356d53ef62eac57c",
                 "is_valid": {
                     "timesteps": [
                         "2022-01-01 00:00:00",
                         "2026-01-01 00:00:00"
                     ],
                     "unit": "N/A",
                     "values": [
-                        true,
-                        true
+                        false,
```

Comment:
Burde vi ikke prøve å ha valid settings?

Reply:
Some of them are valid. Please the the updated snapshot :)